src/apps/ojph_compress/ojph_compress.cpp

```diff
@@ -461,9 +461,9 @@ int main(int argc, char * argv[]) {
 
     "The following option has a default value (optional):\n"
     " -num_decomps  (5) number of decompositions\n"
-    " -qstep        (1.0/255,0) quantization step size; all quantization\n"
-    "               step sizes are derived from this.\n"
-    " -reversible   (false) for irreversible adnd true for reversible\n"
+    " -qstep        (-1.0...1.0) quantization step size; all quantization without 0.0\n"
+    "               step sizes are derived from this. {default -1.0}\n"
+    " -reversible   (false) for irreversible lossy and true for reversible lossless\n"
```

Comment:
-reversible   (false) for irreversible; that is, lossy compression using the 9/7 wavelet transform; and true for reversible compression; that is, lossless compression using the 5/3 wavelet transform.